### PR TITLE
LostUsernameController: handle accounts without a username

### LINKED ISSUE #210: Error in password reset emails for accounts without a username
Hi,

We've had the case where a user had not processed an account invite for a year - and when trying to use her account, she used the password reset feature - and was quite surprised to get an email saying:

    Your username is: null

I can trace this to `virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp`:

````
  <h5>Your username is:</h5>

  ${managedSubject.login.encodeAsHTML()}<br><br>
````

This needs special handling when managedSubject.login is null (e.g., text "Your username has not been set yet.").

Would you be happy to accept a PR fixing this?

Cheers,
Vlad

### PR DESCRIPTION
**Pull request by vladimir-mencl-eresearch**
Fixes #210 - checks for null managedSubject.login and includes a
customized error message.

### CODE REVIEW

File: virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp [hunk 1]
```diff
@@ -10,11 +10,18 @@ We have received a request to provide the username you use for logging in to the
...
 <g:if test="${managedSubject}">
+  <g:if test="${managedSubject.login}">
   <h5>Your username is:</h5>
 
   ${managedSubject.login.encodeAsHTML()}<br><br>
 
   If you have also lost your password, you may initiate a password reset using the <strong>Recover your lost password</strong> link on the AAF Virtual Home login page.
+  </g:if>
+  <g:else>
+  <h5>Your account does not have a usernamet yet.</h5>
```
Comment (bradleybeddoes) [inline virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp]: minor typo here

File: virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp [hunk 2]
```diff
@@ -10,11 +10,18 @@ We have received a request to provide the username you use for logging in to the
...
+
+  This may be because you have not finalized your account invite.  If you cannot find the original email with the account invite, please contact support@aaf.edu.au to request having the account invite sent again.
```
Comment (bradleybeddoes) [inline virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp]: Out of interest how is Tuakiri dealing with support@aaf replacements in the UI when you deploy?

File: virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp (hunk 1 above)
Comment (vladimir-mencl-eresearch) [inline virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp]: Ooops, thanks, fixed!

File: virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp (hunk 2 above)
Comment (vladimir-mencl-eresearch) [inline virtualhome/src/groovy/aaf/vhr/email_lost_username.gsp]: Another commit in our branch to change the branding to Tuakiri.

I could have factored this out to use existing branding ... but decided for the easier minimal-change / higher maintenance route for now...

**Pull request merged**